.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,144 @@
+name: Bug report
+description: Create a report to help us fix something that isn't working as expected
+labels: "bug :bug:"
+body:
+- type: markdown
+  attributes: 
+    value: |
+      ## Before you begin
+      🚨 **Please do not skip instructions**🚨
+      
+      This info is essential for investigating your bug report. Issues with missing information may be closed without investigation.
+      
+      If you're uncertain about the problem, find or start a [Questions & Help](https://github.com/CommunityToolkit/WindowsCommunityToolkit/discussions/categories/questions-help) discussion, where you can get insight from the community regarding your issue. 
+      
+- type: textarea
+  id: description
+  attributes:
+     label: Describe the bug
+     description: A clear and concise description of what the bug is.
+- type: input
+  id: last-working-toolkit-version
+  attributes:
+      label: Regression
+      description: If this is a regression, what toolkit version did you last see it work?
+      placeholder: 
+  validations:
+      required: false
+- type: checkboxes
+  id: reproducable-in-sample-app
+  attributes: 
+    label: Reproducible in sample app?
+    description: Can this be reproduced in the Sample App? (Either in a sample as-is or with new XAML pasted in the editor.)
+    options:
+      - label: This bug can be reproduced in the sample app.
+        required: false
+- type: textarea
```

Comment:
Not all the textareas need to be larger, but this one (for the repro steps) could stand to be larger (by adding `render: text`).